tests/Sdk/Unit/Extension/ValueResolver/PriorityValueResolverTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Unit\Extension\ValueResolver;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Sdk\Core\Application\Dependency\InteractionProcessorInterface;
+use SprykerSdk\Sdk\Extension\Exception\UnresolvableValueExceptionException;
+use SprykerSdk\Sdk\Extension\ValueResolver\PriorityPathValueResolver;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+
+/**
+ * @group Sdk
+ * @group Extension
+ * @group ValueResolver
+ * @group PriorityValueResolverTest
+ */
+class PriorityValueResolverTest extends Unit
```

Comment:
According to the code coverage an important piece of logic isn't covered.
```
        foreach ($this->getSettingPaths() as $settingKey) {
            $path = $settingValues[$settingKey];
            if (strpos($path, DIRECTORY_SEPARATOR, -1) === 0) {
                $path = rtrim($path, DIRECTORY_SEPARATOR);
            }
            $path = implode(DIRECTORY_SEPARATOR, [$path, $relativePath]);
            if (file_exists($path)) {
                return $this->formatValue($path);
            }
        }
```

Please, add additional test/tests to cover the logic inside the `foreach` loop.